content/javascript/concepts/window-prompt/window-prompt.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'prompt()'
+Description: 'Displays a dialog with an optional message asking users to input some text'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`prompt()`** is a method serviced globally by the window object provided by the user's browser.
+
+## Syntax
+
+**`prompt()`** expects 1 to 2 parameters, the first one display the title of the dialog, and the second is used for default value that can also be optional and it is used in case that the user does not input any data, otherwise would be null.
```

Comment:
```suggestion
`prompt()` expects 1 to 2 parameters. The first one is used to display the title of the dialog. The second is used for an optional default value used in case the user does not input any data, otherwise `prompt()` returns null.
```
Only bold the first occurrence of the term in the entry.